rules/repositories.bzl

```diff
@@ -62,10 +62,10 @@ def rules_ios_dependencies():
     _maybe(
         github_repo,
         name = "build_bazel_rules_apple",
-        ref = "942c24380a6cb001d3521c510fdb9b61ee164585",
-        project = "bazelbuild",
+        ref = "f628e6d33e95dd395b07330d7e191f31d6bd525c",
+        project = "dostrander",
```

Comment:
Do we need this fork if https://github.com/bazelbuild/rules_apple/pull/1616 is merged?

Reply:
That is merged but it isn't in the release of 5.x which causes an error of:

```
File "/private/var/tmp/_bazel_derko/45d370c48a980ff7ee8dfaac4184ab30/external/build_bazel_rules_apple/apple/internal/transition_support.bzl", line 528, column 46, in <toplevel>
                _apple_platform_split_transition = transition(
Error in transition: Invalid transition input '//command_line_option:incompatible_enable_apple_toolchain_resolution'. Cannot transition on --experimental_* or --incompatible_* options
ERROR: error loading package 'tests/ios/frameworks/sources-with-prebuilt-binaries': at /Users/derko/Development/rules_ios/rules/apple_patched.bzl:4:5: at /private/var/tmp/_bazel_derko/45d370c48a980ff7ee8dfaac4184ab30/external/build_bazel_rules_apple/apple/apple.bzl:18:5: at /private/var/tmp/_bazel_derko/45d370c48a980ff7ee8dfaac4184ab30/external/build_bazel_rules_apple/apple/apple_static_library.bzl:18:5: initialization of module 'apple/internal/transition_support.bzl' failed
```

When using master. The 5.x must have this fix but not master :/